src/woodbury_pd_mat.jl

```diff
@@ -38,6 +38,8 @@ function WoodburyPDMat(
     return WoodburyPDMat(A, Diagonal(D), Diagonal(S))
 end
 
+WoodburyPDMat{T, TA, TD, TS}(vals...) where {T, TA, TD, TS} = WoodburyPDMat(vals...)
```

Comment:
Why do we need this constructor?

Reply:
Oh, we don't for this MR. Mistake adding this from experimenting with ChainRules. I'll just leave it in though